### PR TITLE
Use bytes32 instead of string

### PR DESCRIPTION
**Pull request by Anboias**
Had to use an encodedDapiName, passing just the dapiName would not work.

image

### CODE REVIEW

File: docs/dapis/developers/data-feed-id-to-reader-to-whitelist-status.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ the hash off-chain of `name` as shown below. Then pass the value of
 `dapiNameHash` as `_datafeedId` in the example above.
 
 ```solidity
-dapiNameHash = ethers.utils.solidityKeccak256(['string'], [dapiName]);
+dapiNameHash = ethers.utils.solidityKeccak256(['bytes32'], [encodedDapiName]);
```
Comment (bbenligiray) [inline docs/dapis/developers/data-feed-id-to-reader-to-whitelist-status.md:62]:
```suggestion
encodedDapiName = ethers.utils.formatBytes32String(dapiName);
dapiNameHash = ethers.utils.solidityKeccak256(['bytes32'], [encodedDapiName]);
```

**Review by bbenligiray — CHANGES_REQUESTED** [review]
It's not clear what `encodedDapiName` is

File: docs/dapis/developers/data-feed-id-to-reader-to-whitelist-status.md (hunk 1 above)
Comment (Anboias) [reply docs/dapis/developers/data-feed-id-to-reader-to-whitelist-status.md:62]: @bbenligiray, I've rephrased the comment and added an additional step where we encode the dapiName. Would this do?

**Review by bbenligiray — APPROVED** [review]
Yes, looks good

**Pull request merged**